### PR TITLE
MNT Remove TODO comments

### PR DESCRIPTION
**Pull request by sabina-talipova**
## Parent issue
- https://github.com/silverstripe/.github/issues/116

### CODE REVIEW

File: code/Controllers/CMSMain.php [hunk 1]
```diff
@@ -80,8 +80,6 @@
  * This class creates a 2-frame layout - left-tree and right-form - to sit beneath the main
  * admin menu.
  *
- * @todo Create some base classes to contain the generic functionality that will be replicated.
- *
```
Comment (sabina-talipova) [inline code/Controllers/CMSMain.php:84]: Open ticket: https://github.com/silverstripe/silverstripe-cms/issues/2896

File: code/Controllers/ContentController.php [hunk 2]
```diff
@@ -45,7 +45,6 @@
  * Subclasses of ContentController are generally instantiated by ModelAsController; this will create
  * a controller based on the URLSegment action variable, by looking in the SiteTree table.
  *
- * @todo Can this be used for anything other than SiteTree controllers?
```
Comment (sabina-talipova) [inline code/Controllers/ContentController.php:48]: I think it's not longer actual issue.

File: code/Controllers/ContentController.php [hunk 3]
```diff
@@ -303,8 +302,6 @@ public function Menu($level)
     /**
      * Returns the default log-in form.
      *
-     * @todo Check if here should be returned just the default log-in form or
-     *       all available log-in forms (also OpenID...)
```
Comment (sabina-talipova) [inline code/Controllers/ContentController.php:307]: Open ticket: https://github.com/silverstripe/silverstripe-cms/issues/2897

File: code/Model/SiteTree.php [hunk 4]
```diff
@@ -1958,7 +1958,6 @@ public function getLiveURLSegment()
      */
     public function BackLinkTracking()
     {
-        // @todo - Implement PolymorphicManyManyList to replace this
         $list = ArrayList::create();
```
Comment (sabina-talipova) [inline code/Model/SiteTree.php:1961]: Open ticket: https://github.com/silverstripe/silverstripe-cms/issues/2895

File: code/Reports/BrokenFilesReport.php
Comment (sabina-talipova) [inline code/Reports/BrokenFilesReport.php:1]: For all `todo: use NestedTitle(2)`, new ticket was created: https://github.com/silverstripe/silverstripe-cms/issues/2894

File: tests/php/Model/SiteTreePermissionsTest.php [hunk 5]
```diff
@@ -12,10 +12,6 @@
 use SilverStripe\Subsites\Extensions\SiteTreeSubsites;
 use SilverStripe\Versioned\Versioned;
 
-/**
- * @todo Test canAddChildren()
- * @todo Test canCreate()
- */
```
Comment (sabina-talipova) [inline tests/php/Model/SiteTreePermissionsTest.php:18]: New ticket: https://github.com/silverstripe/silverstripe-cms/issues/2893

File: tests/php/Search/ZZZSearchFormTest.php [hunk 6]
```diff
@@ -21,9 +21,6 @@
 use TractorCow\Fluent\Extension\FluentSiteTreeExtension;
 
 /**
- * @todo Fix unpublished pages check in testPublishedPagesMatchedByTitle()
- * @todo All tests run on unpublished pages at the moment, due to the searchform not distinguishing between them
- *
  * Because this manipulates the test database in severe ways, I've renamed the test to force it to run last...
```
Comment (sabina-talipova) [inline tests/php/Search/ZZZSearchFormTest.php:24]: I'm not sure. Probably, we can update this class with "TODO" requirements.

File: _config.php [hunk 7]
```diff
@@ -32,7 +32,6 @@
     [SiteTree::class, 'link_shortcode_handler']
 );
 
-// TODO Remove once we can configure CMSMenu through static, nested configuration files
```
Comment (sabina-talipova) [inline _config.php:35]: Since 2012

File: code/Controllers/CMSMain.php
Comment (sabina-talipova) [inline code/Controllers/CMSMain.php:1]: Most of TODO since 2012

**Review by GuySartorelli — APPROVED** [review]
LGTM

**Pull request merged**